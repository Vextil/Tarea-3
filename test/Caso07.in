AgregarProcesos chrome 12 FIN
ImprimirProcesos
EliminarProceso chrome
ImprimirProcesos
Reinicio
AgregarProcesos gedit 93 kdenlive 423 gnome 33 thunderbird 99 FIN
ImprimirProcesos
AgregarProcesos gedit 754 FIN
ImprimirProcesos
EliminarProceso firefox
ImprimirProcesos
ListarProcesos
MuyConsumidores 10
MuyConsumidores 95
MuyConsumidores 900
Fin
