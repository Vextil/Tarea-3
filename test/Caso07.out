1>Hay 2 procesos.
2>
(12,chrome) 
(100,boot) 
3>La memoria que se libera es 12.
4>
(100,boot) 
5>Reiniciando
6>Hay 5 procesos.
7>
(33,gnome) (99,thunderbird) 
(423,kdenlive) 
(93,gedit) 
(100,boot) 
8>Hay 5 procesos.
9>
(33,gnome) (99,thunderbird) 
(423,kdenlive) 
(93,gedit) 
(100,boot) 
10>No existe el proceso.
11>
(33,gnome) (99,thunderbird) 
(423,kdenlive) 
(93,gedit) 
(100,boot) 
12>boot gedit gnome kdenlive thunderbird 
13>boot gedit gnome kdenlive thunderbird 
14>boot kdenlive thunderbird 
15>
16>Fin
